react-native-adjust.podspec

```diff
@@ -0,0 +1,22 @@
+require 'json'
+
+package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
+
+Pod::Spec.new do |s|
+  s.name           = 'react-native-adjust'
+  s.version        = package['version']
+  s.summary        = package['description']
+  s.description    = package['description']
+  s.license        = package['license']
+  s.author         = package['author']
+  s.homepage       = package['homepage']
+  s.source         = { :git => 'https://github.com/adjust/react_native_sdk', :tag => s.version }
+
+  s.requires_arc   = true
+  s.platform       = :ios, '8.0'
+
+  s.preserve_paths = 'LICENSE', 'README.md', 'package.json', 'index.js'
+  s.source_files   = 'ios/*.{h,m}'
```

Comment:
The approach that @tecbot is describing would work for us as long as `s.dependency 'Adjust'` is added to the podspec. For people using `react-native-link`, it would require an "Additional Step for iOS" like the Mixpanel library describes at https://github.com/davodesign84/react-native-mixpanel/tree/0e32006a1184f82d498b2bd0fdd7d3241ced8c5b